common/src/main/java/com/revenuecat/purchases/common/offlineentitlements/OfflineEntitlementsManager.kt

```diff
@@ -71,6 +71,7 @@ class OfflineEntitlementsManager(
                 synchronized(this@OfflineEntitlementsManager) {
                     debugLog(OfflineEntitlementsStrings.UPDATING_OFFLINE_CUSTOMER_INFO_CACHE)
                     _offlineCustomerInfo = customerInfo
+                    deviceCache.getCachedAppUserID()?.let { deviceCache.clearCustomerInfoCache(it) }
```

Comment:
Note that I'm not clearing the ETag cache here. I thought about also clearing that but didn't see any advantages